### DISCUSSION TITLE
Is there a way to easily modify an existing robot (like the Mavic drone)

### DISCUSSION THREAD

**Discussion by thephet** [cyberbotics/webots#6249] [General]
Hi there,

I am wondering if there's an easy way of modifying an existing robot, such as the Mavic drone, and then test how this modification worked.

For example, I would like to change the shape or angle of the blades.

I have opened the Mavic demo, and I have converted the Drone "convert the root to base node" and I see more nodes, but I am not sure if this would be enough.

**Comment by omichel** [comment]
Please ask such technical questions on https://robotics.stackexchange.com with the `webots` tag.

**Reply by thephet** [reply]
Thanks I have done it.

**Reply by omichel** [reply]
I just answered it. Please consider accepting my answer and up-voting it if you believe it is useful.
